app/models/harvest.rb

```diff
@@ -118,4 +138,13 @@ def crop_name_to_human
   def default_photo
     photos.first || crop.default_photo
   end
+
+  def update_predictions
+    return if planting.blank?
+    # Set the actual harvest on the planting
+    planting.update(harvested_at: Harvest.where(planting: self).order(:harvested_at).first.harvested_at)
```

Comment:
I'm really confused by this line. This is a method on `Harvest`, so `self` should be a Harvest, right? In which case `Harvest.where(planting: self)` makes no sense. Shouldn't this be something on the lines of `planting.update(harvested_at: harvested_at) if planting.harvested_at.blank?`